src/node_env_var.cc

```diff
@@ -213,19 +181,19 @@ std::shared_ptr<KVStore> KVStore::Clone(v8::Isolate* isolate) const {
   for (uint32_t i = 0; i < keys_length; i++) {
     Local<Value> key = keys->Get(context, i).ToLocalChecked();
     CHECK(key->IsString());
-    copy->Set(isolate, key.As<String>(), Get(isolate, key.As<String>()));
+    copy->Set(isolate, key.As<String>(), Get(isolate, key.As<String>())
+        .ToLocalChecked());
```

Comment:
nit: the indentation here is a bit misleading, it looks as if `.ToLocalChecked()` is called on the return value of `Set()`.

You can use `CLANG_FORMAT_START=$START make format-cpp` to format the style where `$START` is the commit ID before the first commit of this PR (or `=master` if your local master is still where your local PR branch starts).

Reply:
Thanks Joyee Cheung for sharing an elegant way to format the code file, the changes are committed at https://github.com/nodejs/node/pull/27310/commits/34c8538aef50d3a392dc4d6522c472f57cc2472e